### DISCUSSION TITLE
Planned new simulations

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#59] [General]
Suggested new simulation(s):
“Runners”: @monsieuralok , @mvdebolskiy , @adagj , @kjetilaas , and @tjiputra 

- 1st: albedo in CLM-FATES (https://github.com/NorESMhub/noresm3_dev_simulations/issues/56)
- 2nd: CAM + albedo
- 3rd: CAM + albedo + gust wind param.
- 4th: CAM + albedo + COARE 
- 5th: DATM for spinning up the ocean

@mvertens @rosiealice @gold2718

**Comment by monsieuralok** [comment]
@mvertens Can we describe 3rd simulation Compset, CAM version and namelist or anything we want to change in alpha07 tag? Then, I can plan to run it.

**Comment by mvertens** [comment]
@adagj - in terms of 2-5 - I thought we were going to use the newest version of CAM that is still not quite ready. Or were we going to stick with the original version currently used in alpha07. I thought we were aiming for the new version which is why I have not described this simulation yet.

**Comment by mvertens** [comment]
@adaj @alak @rosiealice @gold2718 - My understanding was that we would do the following. Is there agreement on this?

2nd: noresm2_5_alpha08(CAM) + 1st
3rd: 2nd + gust_wind param
4th: 2nd + COARE
5th: DATM for spinning up ocean

**Reply by adagj** [reply]
Yes, that sounds like a good plan. And thanks for clarifying, I forgot to put tags and such
